### PR TITLE
Code Review by TSE

### PR DESCRIPTION
**Pull request by dxania**

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi team!

### Nice job! Project looks! :+1: 

### But first of all let's fix the next problems and we can move on.

#### Page structure

##### Navigation bar

* [ ] The search bar should be is aligned to the middle of the navbar 
* [ ] The search button should be at the end (rightmost part) of the search bar 
* [ ] It should has at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar 

Screenshot from 2019-11-29 02-14-15

##### Video

* [ ] It has a share button 


* Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project needs changes

File: index.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>YouTube Video Player Clone</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="CSS/styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
```
Comment (Anna-Myzukina) [inline index.html]: The value of attribute [ href ] must be in double quotes.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+            <div id="aboutvideoleft">
+                <h5>
+                    634 views •
+                    Nov 26, 2019
+                </h5>
+            </div>
+            <div id="aboutvideoright">
+                <i class="fa fa-thumbs-up" aria-hidden="true">&nbsp;26K</i>
+                <i class="fa fa-thumbs-down" aria-hidden="true">&nbsp;720</i>
+                <i class="fa fa-mail-forward" aria-hidden="true">&nbsp;SHARE</i>
```
Comment (pelzolga123) [inline index.html]: 'Share' should be a button.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
One more change required.

**Comment by pelzolga123** [comment]
## Hi Team,
### Great  work :+1: Only one thing needs to be changed before project can be approved.
* [ ] Share should be a button, but now it is an icon.
image

[Optional] You can make Readme file more descriptive and add live version of your project using [ this link](http://raw.githack.com/). 

### Please kindly Fix the above issue and send another code review request.
If you have any questions or need help, you can message me on slack. *Happy coding*

**Review by MauricioRobayo — APPROVED** [review]
Great job fixing everything. Go ahead and continue 🚀  .

Status: **Approve**

**Pull request merged**